calcite-qml/Calcite/Calcite.qml

```diff
@@ -18,7 +18,8 @@ pragma Singleton
 import QtQuick 2.0
 
 /*!
- \qmltype Calctie
+ \qmltype Calcite
+ \inqmlmodule Esri.ArcGISRuntime.Toolkit
```

Comment:
❌  This isn't a Toolkit module though! If anything, it's in its own Calcite module. If there's a way to blocklist the Calcite folder from being documented that would be better.

Reply:
Remove the qdocs commits from this file?

Reply:
I wouldn't remove the comments, they are valuable, and may not remove the warnings.

In the `ArcGISQtToolkit.qdocconf` file which is used to build the toolkit there is an `excludedirs` property which looks like this:

```conf
# do not process platform-specific AR headers or examples
excludedirs += ../../toolkit/augmentedreality/Common/include/iOS \
               ../../toolkit/augmentedreality/Common/include/Android \
               ../../toolkit/augmentedreality/Examples
```

which can also have this added:

```conf
# do not process platform-specific AR headers or examples
excludedirs += ../../toolkit/augmentedreality/Common/include/iOS \
               ../../toolkit/augmentedreality/Common/include/Android \
               ../../toolkit/augmentedreality/Examples \
               ../../toolkit/calcite-qml
```